app/models/bot.rb

```diff
@@ -12,8 +12,36 @@ class Bot < ActiveRecord::Base
   validates :hair_color, presence: true
   validates :user_id, presence: true
 
+  WELCOME_MESSAGE = 'Yeiii!'
+  UNKNOWN_COMMAND_MESSAGE = "This is a test"
+
   def avatar
     gender + '-' + hair_color + '-' + eye_color + '.png'
   end
 
+  def send_welcome_message
+    self.send_message(self.user.full_phone_number,WELCOME_MESSAGE)
+  end
+
+  def send_message(recipient_phone,body)
+    @twilio_number = ENV['TWILIO_PHONE_NUMBER']
+    client = Twilio::REST::Client.new ENV['TWILIO_ACCOUNT_SID'], ENV['TWILIO_AUTH_TOKEN']
```

Comment:
Line is too long. [89/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.